test/eradius_lib_SUITE.erl

```diff
@@ -0,0 +1,48 @@
+%    __                        __      _
+%   / /__________ __   _____  / /___  (_)___  ____ _
+%  / __/ ___/ __ `/ | / / _ \/ / __ \/ / __ \/ __ `/
+% / /_/ /  / /_/ /| |/ /  __/ / /_/ / / / / / /_/ /
+% \__/_/   \__,_/ |___/\___/_/ .___/_/_/ /_/\__, /
+%                           /_/            /____/
+%
+% Copyright (c) Travelping GmbH <info@travelping.com>
```

Comment:
eradius is under MIT license, please use a complete license header when adding a new file (I know that this is not strictly required for MIT, but it look more complete).